### ISSUE TITLE
Callback functions?

### ISSUE THREAD

**Issue by macsupport** [1rosehip/jplist-es6#30]
I'm not seeing any documentation for callback functions. In jquery plist there was `redrawCallback `. Any similar functions for this version? I have a function that fails on pagination page change and needs to be called as each page is shown.

**Comment by 1rosehip** [comment]
Yes, redrawCallback function is missing. i'm going to add it in the next update.

**Comment by Fitomi** [comment]
Facing the same issue, any update on this?
Thanks.

**Comment by s3am** [comment]
Hello
Any news about Callback functions implementation ?
Best regards

**Comment by jameshobden** [comment]
Also interested in this feature please!

**Comment by jameshobden** [comment]
Hi Guys, I have implemented a workaround by listening to the state event, as documented here: https://www.jplist.org/documentation/api/jplist-state-event
